src/Camera.cc

```diff
@@ -228,8 +228,11 @@ Errors Camera::Load(ElementPtr _sdf)
   this->dataPtr->triggered = _sdf->Get<bool>("triggered",
       this->dataPtr->triggered).first;
 
-  this->dataPtr->triggerTopic = _sdf->Get<std::string>("trigger_topic",
-      this->dataPtr->triggerTopic).first;
+  if (_sdf->HasElement("trigger_topic"))
+  {
+    this->dataPtr->triggerTopic = _sdf->Get<std::string>("trigger_topic",
```

Comment:
I understood we shouldn't need to call `HasElement` before using the `Get` call that passes a default value. If we call `HasElement`, how will it ever get to use the default value?

It looks like the use of `__default__` was intentional, judging by the documentation, should that be updated too?

https://github.com/ignitionrobotics/sdformat/blob/12e77821af7c2810c4d9d532ee8cb833cbb8e318/sdf/1.9/camera.sdf#L12-L14

Reply:
I think I used `__default__/trigger` since a camera with no name is given the name `__default__`. I made it so that */trigger uses the name of the camera to create the prefix (i.e. /camera123 -> /camera123/trigger).

There is probably a better way to handle the camera name rather than to use `__default__/trigger`.

https://github.com/ignitionrobotics/sdformat/blob/12e77821af7c2810c4d9d532ee8cb833cbb8e318/sdf/1.9/camera.sdf#L4-L6

Reply:
I think the default value of `trigger_topic` should be an empty string and leave the behavior up to a downstream library (`ign-sensors`). I will update the documentation accordingly.

Reply:
https://github.com/ignitionrobotics/sdformat/pull/1006/commits/2948c0932465768abc2dc8e247beedad16fd41de